### DISCUSSION TITLE
Triton use user-vert-functions.glsl  error

### DISCUSSION THREAD

**Discussion by lyh19930821** [pelicanmapping/osgearth#2571] [Support Q&A] [answered]
osg365+osgearth3.2  use `user-vert-functions.glsl`  hase error.

![image](https://github.com/gwaldron/osgearth/assets/58672894/034ea1f9-d165-4cca-b873-8ec36dd11f3b)

I can't see the Triton ocean effect.

English is not my first language. This is machine translated.

Thank.

**Comment by gwaldron** [comment]
Please try osgEarth 3.5 with latest version of Triton.

**Comment by lyh19930821** [comment, marked as answer]
I update osgEarth 3.5 . I can see Triton ocean effect. Thanks.

![5f0481046e663ef4921c9da78c0b09b](https://github.com/gwaldron/osgearth/assets/58672894/bff76884-b92b-44f5-b313-84ed3ba5ae19)
